README.MD

```diff
@@ -4,15 +4,98 @@ This Github action allows to handle Dependabot pull requests.
 
+💡 Interactions with Dependabot are done through a [Github personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). It means that the action will answer and give commands to dependabot **on the behalf of the token's owner**.
+
 ## Contributions and development
 
 📚 For development documentation, read the [Contributing](https://github.com/fewlinesco/dependabot-auto-merge/blob/main/CONTRIBUTING.md) file.
 
+## Usage
+
+In order to add this action to a repository, you need to create a new workflow:
+
+```yaml
+# .github/workflows/auto-merge.yaml
+name: AutoMerge
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+
+    name: Approve and ask for merge
+
+    # It is advised to add this line in order to totally skip the code execution
+    # in case Dependabot is not the requester.
+    if: ${{ github.actor == 'dependabot[bot]' }}
+
+    steps:
+      - name: 🔬 Auto merge
+        uses: fewlinesco/dependabot-auto-merge
+        with:
+          github-token: ${{ secrets.GH_TOKEN }}
+```
+
+This basic configuration will allow **any pull request from Dependabot** to be approved and merged.
+
+### Workflow inputs
+
+#### `github-token`
+
+```yaml
+# ...
+- name: 🔬 Auto merge
+  uses: fewlinesco/dependabot-auto-merge
+  with:
+    github-token: ${{ secrets.GH_TOKEN }}
+```
+
+The `github-token` input is **required**. It is a Github PAT and therefore allows the Github action to act as a user.
+For this Github action, the **required token's scopes are `read:org, repo, workflow, write:discussion`**.
+
+It's also required to **register github token into Dependabot's secrets** on the Repository (*⚙️ settings > secrets > dependabot*).
+
+#### `npm-disallowlist` and `gha-disallowlist`
+
+A list of dependancies on which you want additional control. The syntax is `<dependency-name>[:<limitation>]`:
+
+
+```yaml
+# ...
+- name: 🔬 Auto merge
+  uses: fewlinesco/dependabot-auto-merge
+  with:
+    github-token: ${{ secrets.GH_TOKEN }}
+    npm-disallowlist: pkg-a:major pkg-b:minor
+    gha-disallowlist: >
+      pkg-c:patch
+      pkg-d pkg-e*"
+```
+
+> one line and multine syntaxes are allowed.
+
+In this example:
+- `pkg-a:major` means major bumps won't be allowed for `pkg-a`. But minors and patches can still be auto merged.
+- `pkg-b:minor` means major and minor bumps won't be allowed for `pkg-b`. But patches can still be auto merged.
+- `pkg-c:patch` and `pkg-d` is a different syntax for the same thing: no auto merge allowed.
+- `pkg-e*` means that every package with a name starting with `pkg-e` will be impacted. It is cumulative with bump level. e.g: we could want to freeze every `"@typescript-eslint` on a major version: `"@typescript-eslint/*:major`.
```

Comment:
Maybe we  should add that it only works with `*` at the end? It's what I understood the code to do